slam_vdb/include/slam_vdb/slam_vdb.hpp

```diff
@@ -4,4 +4,14 @@
 #include "slam_vdb/measurement/point/point_xyz.hpp"
 #include "slam_vdb/measurement/pointcloud/pointcloud.hpp"
 
+namespace slam_vdb {
+class slam_vdb
+{
+private:
+  /* data */
+public:
+  slam_vdb(/* args */);
+  ~slam_vdb();
+};
+}  // namespace slam_vdb
```

Comment:
I don't think we need to add this. we can still create header-only library and install in our system.

Reply:
I create this to generate a `.so` and link to the ROS node. Is the entire library gonna be header-only in the future?